nikola/utils.py

```diff
@@ -1673,6 +1674,31 @@ def escape(s):
     return '/'.join([escape(p) for p in category_path])
 
 
+def colorize_str_from_base_color(string, base_color):
+    """Find a perceptual similar color from a base color based on the hash of a sring.
```

Comment:
string*

also, it would be nice to expose this function as part of the global context